sdk/identity/identity/test/internal/identityClient.spec.ts

```diff
@@ -55,7 +55,7 @@ describe("IdentityClient", function() {
       mockHttp.tokenCredentialOptions
     );
     await assertRejects(credential.getToken("https://test/.default"), (error) => {
-      assert.strictEqual(error.name, "AuthenticationError");
+      assert.ok(error instanceof CredentialUnavailable);
```

Comment:
Strongly prefer name checks to `instanceof` since the latter is unreliable.

Reply:
I'll resolve this since I'll address it with the next things I'll push today.